src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/_moreDetails.html.twig

```diff
@@ -1,8 +1,10 @@
+{% import '@SyliusUi/Macro/flags.html.twig' as flags %}
+
 <div id="more-details" class="ui styled fluid accordion">
     {% for translation in product.translations %}
         <div class="title">
             <i class="dropdown icon"></i>
-            <i class="{{ translation.locale|slice(-2)|lower }} flag"></i>
+            {{ flags.fromCountryCode(translation.locale|sylius_locale_country|lower) }}
```

Comment:
There's a  `fromLocaleCode` macro that returns the flag based on locale code, if possible.

```suggestion
            {{ flags.fromLocaleCode(translation.locale) }}
```